src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -896,23 +909,15 @@ public JiraIssue getIssue(final String id) throws IOException {
         }
     }
 
-    @Deprecated
```

Comment:
Leave the deprecated things for a major release (where they will be cleaned up).